frontend/views/components/group-creation-steps/GroupMincome.vue

```diff
@@ -38,27 +38,66 @@
 
       i18n.helper This value can be adjusted in the future.
 
+    fieldset.field
+      .label.c-label-tooltip
+        i18n In what day should the first payment distribution be calculated?
+        tooltip(direction='bottom-end')
+          span.button.is-icon-small
+            i.icon-question-circle
+          template(slot='tooltip')
+            i18n(tag='p') Select which day of the month should the distribution be calculated. Every group member will need to update their mincome details before this date to ensure that the algorithm can fairly distribute available funds between group members.
+
+      .selectbox
+        select.select(
+          :aria-label='L("Choose day")'
+          name='distributionDate'
+          required=''
+          :value='group.distributionDate'
+          @change='update'
+        )
+          i18n(tag='option' disabled value='') Choose your group's distribution date
+          option(
+            v-for='(item, index) in ephemeral.distributionDayRange'
+            :key='index'
+            :value='item'
+          ) {{ humanDate(item, { month: 'long', year: 'numeric', day: 'numeric' }) }}
+
+      i18n.helper Payment distribution will be calculated every 30 days.
     slot
 </template>
 
 <script>
 import currencies from '@view-utils/currencies.js'
+import Tooltip from '@components/Tooltip.vue'
+import { dateToPeriodStamp, addTimeToDate, DAYS_MILLIS, humanDate } from '~/frontend/utils/time.js'
 
 export default ({
   name: 'GroupMincome',
+  components: {
+    Tooltip
+  },
   props: {
     group: { type: Object },
     $v: { type: Object }
   },
   data () {
     return {
-      currencies
+      currencies,
+      ephemeral: {
+        distributionDayRange: []
+      }
+    }
+  },
+  beforeMount () {
+    for (let index = 0; index < 30; index++) {
```

Comment:
I noticed an issue in my testing, which is that with this method (at least in my timezone), the previous day will show up in the dropdown menu.

So if today is May 3rd, then May 2nd will show up.

To fix, could you update this to the following?

```js
for (let index = 1; index <= 30; index++) {
```